src/hammer-vlsi/hammer_vlsi/hammer_vlsi_impl.py

```diff
@@ -706,6 +706,11 @@ def signoff_results(self) -> int:
 
 class HammerDRCTool(HammerSignoffTool):
 
+    def export_config_outputs(self) -> Dict[str, Any]:
```

Comment:
These are fine though at the moment we don't really use things like `drc-output.json` unless I missed a memo...?

Reply:
They aren't, but on the off chance they are consumed by something it'll be less confusing to have things be consistent. Also, it makes the Makefile a lot more uniform looking (the output jsons are the targets in the Makefile)